lib/repl.js

```diff
@@ -150,7 +150,6 @@ function REPLServer(prompt,
   self.ignoreUndefined = !!ignoreUndefined;
   self.replMode = replMode || exports.REPL_MODE_SLOPPY;
   self.underscoreAssigned = false;
-  self.last = undefined;
```

Comment:
Well, I suppose only in that it's unnecessary to explicitly assign `self.last = undefined` because in user-land, `myServer.last` will be undefined no matter what, until it becomes assigned. I can revert this change if you think keeping it would allow for more clarity.